opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -46,7 +46,7 @@ namespace Opm {
          const double gravity_arg,
          const std::vector<double>& depth_arg,
          const std::vector<double>& pv_arg,
-         const RateConverterType* rate_converter,
+         RateConverterType* rate_converter,
```

Comment:
I'd prefer to keep this const (and that a reference was used). Then the model class should receive a non-const reference as a constructor argument instead, and this class' rateConverter() method can be removed.